edifikana/front-end/shared-compose/src/commonMain/kotlin/com/cramsan/edifikana/client/lib/features/root/auth/signup/SignUpScreen.kt

```diff
@@ -79,47 +107,86 @@ internal fun SignInV2Content(
         contentAlignment = Alignment.Center,
     ) {
         Column(
-            modifier = Modifier.padding(Padding.MEDIUM),
+            modifier = Modifier
+                .sizeIn(maxWidth = Size.COLUMN_WIDTH)
+                .padding(Padding.MEDIUM),
             verticalArrangement = Arrangement.spacedBy(Padding.MEDIUM),
             horizontalAlignment = Alignment.CenterHorizontally,
         ) {
-            uistate.signUpForm.errorMessage?.let {
-                Text(it)
+            AnimatedContent(
+                uistate.signUpForm.errorMessage,
+                transitionSpec = {
+                    fadeIn()
+                        .togetherWith(
+                            fadeOut()
+                        )
+                },
+            ) {
+                val showErrorMessage = it.isNullOrBlank().not()
+                if (showErrorMessage) {
+                    Text(
+                        it.orEmpty(),
+                        style = MaterialTheme.typography.labelMedium,
+                        color = MaterialTheme.colorScheme.error,
+                    )
+                }
             }
+
             TextField(
                 value = uistate.signUpForm.fullName,
-                onValueChange = {},
-                label = {Text("Full Name (required)")},
+                onValueChange = { onFullNameValueChange(it) },
+                label = { Text(stringResource(Res.string.sign_up_screen_text_full_name)) },
```

Comment:
Set the label from the string resource and then hook up the value change callback.